src/frontend/src/components/ChangeRequestDetailCard.tsx

```diff
@@ -3,97 +3,173 @@
-        </Typography>
-      </Grid>
-    </Grid>
+    <Box
+      sx={{
+        backgroundColor: theme.palette.divider,
+        width: '100%',
+        minHeight: 75,
+        borderRadius: 1,
+        marginTop: 1,
+        paddingTop: 0.5,
+        paddingLeft: 1,
+        paddingRight: 1,
+        paddingBottom: 0.5
+      }}
+    >
+      <Typography variant="body1" fontSize={14} noWrap>
+        {DescriptionType + ': '}
+        <Link color="inherit" component={RouterLink} to={`${routes.PROJECTS}/${wbsPipe(cr.wbsNum)}`}>
+          {wbsPipe(cr.wbsNum)}
+        </Link>
+      </Typography>
+    </Box>
   );
 };
 
-const StageGateCardDetails = ({ cr }: { cr: StageGateChangeRequest }) => {
+const StandardCardDetails = ({ cr }: { cr: StandardChangeRequest }) => {
+  const theme = useTheme();
   return (
-    <Box ml={-1}>
-      {cr.confirmDone ? (
-        <Chip icon={<DoneAll />} label={'Done'} sx={{ backgroundColor: 'transparent' }} />
-      ) : (
-        <Chip icon={<Cancel />} label={'Not Done'} sx={{ backgroundColor: 'transparent' }} />
-      )}
+    <Box
+      sx={{
+        backgroundColor: theme.palette.divider,
+        width: '100%',
+        minHeight: 75,
+        borderRadius: 1,
+        marginTop: 1,
+        paddingTop: 0.5,
+        paddingLeft: 1,
+        paddingRight: 1,
+        paddingBottom: 0.5
+      }}
+    >
+      {cr.what}
     </Box>
   );
 };
 
-const ActivationCardDetails = ({ cr }: { cr: ActivationChangeRequest }) => {
+const ChangeRequestTypePill = ({ type }: { type: ChangeRequestType }) => {
   return (
-    <Box>
-      <Stack direction="row">
-        <Chip
-          icon={<Construction />}
-          label={fullNamePipe(cr.projectLead)}
-          sx={{ backgroundColor: 'transparent', mr: 2, ml: -1, maxWidth: '150' }}
-        />
-        <Chip
-          icon={<Work />}
-          label={fullNamePipe(cr.projectManager)}
-          sx={{ backgroundColor: 'transparent', maxWidth: '150' }}
-        />
-      </Stack>
-      <Stack direction="row" justifyContent={'space-between'}>
-        <Chip icon={<Start />} label={datePipe(cr.startDate)} sx={{ backgroundColor: 'transparent', ml: -1 }} />
-      </Stack>
-    </Box>
+    <Chip
+      size="small"
+      label={ChangeRequestTypeTextPipe(type)}
+      variant="filled"
+      sx={{
+        fontSize: 12,
+        color: 'white',
+        backgroundColor: red[600],
+        mb: 0.5,
+        width: 100
+      }}
+    />
+  );
+};
+
+const ChangeRequestStatusPill = ({ status }: { status: ChangeRequestStatus }) => {
+  const statusPillColor = determineChangeRequestStatusPillColor(status);
+  return (
+    <Chip
+      size="small"
+      label={ChangeRequestStatusTextPipe(status)}
+      variant="filled"
+      sx={{
+        fontSize: 12,
+        color: 'white',
+        backgroundColor: statusPillColor,
+        mb: 0.5,
+        width: 100
+      }}
+    />
+  );
+};
+
+const ChangeRequestCardHeader = ({ cr }: { cr: ChangeRequest }) => {
```

Comment:
I'd put this code in the top-level component, along with the code in ChangeRequestSubWBSDetails, just since there are already so many other custom components in this file.

Reply:
Is there a negative effect in creating custom components other than just having components everywhere?

Reply:
Generally I would say no, my main concern here was readability of the code